docs/guide/tech-behind-calypso.md

```diff
@@ -2,69 +2,21 @@
 
 Understanding the technologies and abstractions on which Calypso is built can make for a much better learning and building experience. Even if you are a student of the “learning by doing” school, we would still encourage you spend some time with the resources in this document.
 
-## No Big Famous Framework
+## Based on React
 
-Calypso is using neither Angular nor Ember; because we are building Calypso for the long haul, updating and improving a home-grown framework is a better long-term approach for us. Currently, we're using React and Flux/Redux; they're great, but knowing we have the control to use better technologies as they come along makes us feel more confident in our future. Calypso isn't a small startup project; we know it will need to scale and our technology will need to scale with it.
+Calypso is composed of multiple applications that are all built using React. [React](http://facebook.github.io/react/) is a library by Facebook that implements a virtual DOM. Instead of carefully changing the DOM, we can just re-render whole components when the data changes. React radically simplified our code and allowed for an awesome composition of reusable components.
 
-## Modern Modular JavaScript
+- [Official documentation](https://react.dev/learn)
```

Comment:
The official document is way better that it used to be, and it contains all what's needed to be honest.